`jbang wrapper install` throws NPE  @v0.88.0

**Describe the bug**
`jbang wrapper install` crashes with a NPE:
```
✗ jbang wrapper install --force --verbose
[jbang] jbang version 0.88.0
[jbang] [ERROR] Cannot invoke "java.nio.file.Path.getFileSystem()" because "path" is null
java.lang.NullPointerException: Cannot invoke "java.nio.file.Path.getFileSystem()" because "path" is null
        at java.base/java.nio.file.Files.provider(Files.java:105)
        at java.base/java.nio.file.Files.isDirectory(Files.java:2316)
        at dev.jbang.cli.Wrapper.install(Wrapper.java:28)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1972)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2346)
        at dev.jbang.cli.JBang$3.handle(JBang.java:145)
        at dev.jbang.cli.JBang$3.handle(JBang.java:140)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at dev.jbang.Main.main(Main.java:14)
[jbang] If you believe this a bug in jbang, open an issue at https://github.com/jbangdev/jbang/issues
```

**To Reproduce**
Steps to reproduce the behavior:
1. Install the v0.88.0
2. run `jbang wrapper install`

**Expected behavior**
The wrapper is created without errors.

**JBang version**
[jbang] jbang version 0.88.0
Cache: /Users/jordisola/.jbang/cache
Config: /Users/jordisola/.jbang
Repository:/Users/jordisola/.m2/repository
0.88.0

**Additional context**
The same fails for both Mac and Linux versions.

--- a/docs/modules/ROOT/pages/configuration.adoc
+++ b/docs/modules/ROOT/pages/configuration.adoc
@@ -42,7 +42,7 @@ classpath:/jbang.properties
    init.template = hello
    run.debug = 4004
    run.jfr = filename={baseName}.jfr
-   wrapper.dir = .
+   wrapper.install.dir = .
 ```
 
 

--- a/itests/wrapper.feature
+++ b/itests/wrapper.feature
@@ -29,3 +29,8 @@ When command('jbang wrapper install -d ' + scratch)
   * match exit == 0
   * command('jbang wrapper install -f -d ' + scratch)
   * match exit == 0
+
+Scenario: test plain wrapper install
+When command('jbang wrapper install')
+  * match exit == 0
+

--- a/src/main/resources/jbang.properties
+++ b/src/main/resources/jbang.properties
@@ -1,4 +1,4 @@
 init.template=hello
 run.debug=4004
 run.jfr=filename={baseName}.jfr
-wrapper.dir=.
+wrapper.install.dir=.

--- a/src/test/java/dev/jbang/cli/TestConfig.java
+++ b/src/test/java/dev/jbang/cli/TestConfig.java
@@ -47,7 +47,7 @@ public class TestConfig extends BaseTest {
 						"run.jfr = filename={baseName}.jfr\n" +
 						"three = baz\n" +
 						"two = bar\n" +
-						"wrapper.dir = .\n"));
+						"wrapper.install.dir = .\n"));
 	}
 
 	@Test

